rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stat_cfg-v0.0.1.json

```diff
@@ -0,0 +1,175 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Statistics report configuration",
+  "description": "JSON schema for statistics report configuration.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "indicator",
+    "owner",
+    "organisation",
+    "category",
+    "frequency",
+    "description",
+    "name",
+    "status"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate stats cfg records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://bib.rero.ch/schemas/stats_cfg/stat_cfg-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Stat ID.",
+      "type": "string",
+      "minLength": 1
+    },
+    "indicator": {
+      "title": "Statistics indicator",
```

Comment:
If possible add a "description" to have a human readable description of fields (ask to PO if needed).
Without description I can't understand the difference between "period" and "frequency" by exemple